<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cloth Simulation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="s_mainWindowGridLayout">
    <item row="0" column="1">
     <widget class="QGroupBox" name="s_clothGB">
      <property name="title">
       <string>Cloth</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="6" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Gravity</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="m_dampingConstant">
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>128.000000000000000</double>
         </property>
         <property name="value">
          <double>512.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="m_clothHeightRes">
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="value">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Height</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Height Resolution</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Width</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Width Resolution</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QCheckBox" name="m_applyWind">
         <property name="text">
          <string>Apply Wind</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="m_clothHeight">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>8.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.636000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Damping Constant</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Spring Constant</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QDoubleSpinBox" name="m_clothWidth">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>8.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>2.560000000000000</double>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QCheckBox" name="m_applySphereCollision">
         <property name="text">
          <string>Apply Sphere Collision</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QSpinBox" name="m_clothWidthRes">
         <property name="suffix">
          <string/>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="value">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QDoubleSpinBox" name="m_springConstant">
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>128.000000000000000</double>
         </property>
         <property name="value">
          <double>1024.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QCheckBox" name="m_applySelfCollision">
         <property name="text">
          <string>Apply Self Collision</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Simulation Speed</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0" colspan="2">
        <widget class="QPushButton" name="m_resetButton">
         <property name="text">
          <string>Reset Cloth</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QCheckBox" name="m_anchorBottomLeft">
         <property name="text">
          <string>Bottom Left</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QCheckBox" name="m_paused">
         <property name="text">
          <string>Paused</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Anchored Corners</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QCheckBox" name="m_anchorBottomRight">
         <property name="text">
          <string>Bottom Right</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QCheckBox" name="m_anchorTopRight">
         <property name="text">
          <string>Top Right</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QCheckBox" name="m_anchorTopLeft">
         <property name="text">
          <string>Top Left</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QDoubleSpinBox" name="m_gravity">
         <property name="minimum">
          <double>-256.000000000000000</double>
         </property>
         <property name="maximum">
          <double>256.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>4.000000000000000</double>
         </property>
         <property name="value">
          <double>32.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QDoubleSpinBox" name="m_simSpeed">
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>m_applyWind</zorder>
      <zorder>m_clothWidth</zorder>
      <zorder>m_clothHeight</zorder>
      <zorder>label_4</zorder>
      <zorder>label_5</zorder>
      <zorder>m_clothWidthRes</zorder>
      <zorder>m_clothHeightRes</zorder>
      <zorder>label_6</zorder>
      <zorder>label_7</zorder>
      <zorder>m_applySphereCollision</zorder>
      <zorder>m_springConstant</zorder>
      <zorder>m_dampingConstant</zorder>
      <zorder>label_8</zorder>
      <zorder>label_9</zorder>
      <zorder>m_applySelfCollision</zorder>
      <zorder>m_paused</zorder>
      <zorder>label_10</zorder>
      <zorder>m_anchorTopLeft</zorder>
      <zorder>m_anchorTopRight</zorder>
      <zorder>m_anchorBottomLeft</zorder>
      <zorder>m_anchorBottomRight</zorder>
      <zorder>m_resetButton</zorder>
      <zorder>label_11</zorder>
      <zorder>label_12</zorder>
      <zorder>m_gravity</zorder>
      <zorder>m_simSpeed</zorder>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="s_drawGB">
      <property name="title">
       <string>Draw</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QComboBox" name="m_drawType">
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string>Triangles</string>
         </property>
         <item>
          <property name="text">
           <string>Triangles</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Lines</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Points</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="m_wireframe">
         <property name="text">
          <string>Wireframe</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="3">
     <widget class="QFrame" name="m_frame">
      <property name="minimumSize">
       <size>
        <width>768</width>
        <height>640</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
